### PR TITLE
Gruppe 1

### PR DESCRIPTION
**Pull request by chrisguter**

### CODE REVIEW

File: src/main/java/de/cas/mse/exercise/csv/CsvReader.java [hunk 1]
```diff
@@ -1,16 +1,55 @@
...
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
 
 import de.cas.mse.exercise.csv.ui.CsvUi;
 
 public class CsvReader {
+
+    private static String COMMA_SEPARATOR = ",";
```
Comment (hpap) [inline src/main/java/de/cas/mse/exercise/csv/CsvReader.java:16]: missing final?

File: src/main/java/de/cas/mse/exercise/csv/CsvReader.java [hunk 2]
```diff
@@ -1,16 +1,55 @@
...
+    private  String currentRow = "";
```
Comment (hpap) [inline src/main/java/de/cas/mse/exercise/csv/CsvReader.java:17]: wofür ist diese variable?

Comment (hpap) [reply src/main/java/de/cas/mse/exercise/csv/CsvReader.java:17]: besser als lokale variable in der methode

File: src/main/java/de/cas/mse/exercise/csv/CsvReader.java [hunk 3]
```diff
@@ -1,16 +1,55 @@
...
+        while(currentRow != null) {
+            currentRow = fileReader.readLine();
+            String[] values = currentRow.split(COMMA_SEPARATOR);
+            dataRows.add(Arrays.asList(values));
+        }
+        
+    }
+
+    private void setHeader() {
+        csvUi.addRow(dataRows.get(0));
```
Comment (hpap) [inline src/main/java/de/cas/mse/exercise/csv/CsvReader.java:45]: maybe 0 as constant

File: src/main/java/de/cas/mse/exercise/csv/CsvReader.java [hunk 4]
```diff
@@ -1,16 +1,55 @@
...
+    }
+
+    private void setRows() {
+        while () {
+            csvUi.addRow(dataRows.get(0));
```
Comment (hpap) [inline src/main/java/de/cas/mse/exercise/csv/CsvReader.java:50]: ;( what happened here?

Comment (hpap) [reply src/main/java/de/cas/mse/exercise/csv/CsvReader.java:50]: obviously not finished:

- index 0 should be header
- while-loop not complete

File: src/main/java/de/cas/mse/exercise/csv/CsvReader.java [hunk 5]
```diff
@@ -1,16 +1,55 @@
...
+        }
+    }
 	
 	public void setCsvUi(final CsvUi csvUi) {
```
Comment (hpap) [inline src/main/java/de/cas/mse/exercise/csv/CsvReader.java:54]: public methods in the beginning

File: src/main/java/de/cas/mse/exercise/csv/CsvReader.java (hunk 1 above)
Comment (Inusette) [inline src/main/java/de/cas/mse/exercise/csv/CsvReader.java:16]: did you forget final? :)

File: src/main/java/de/cas/mse/exercise/csv/CsvReader.java [hunk 6]
```diff
@@ -1,16 +1,55 @@
...
+        }
+        
+    }
+
+    private void setHeader() {
+        csvUi.addRow(dataRows.get(0));
+    }
+
+    private void setRows() {
+        while () {
```
Comment (Inusette) [inline src/main/java/de/cas/mse/exercise/csv/CsvReader.java:49]: ooops? Does this compile?

**Pull request merged**